services/src/main/java/org/jboss/windup/web/services/rest/MigrationProjectEndpointImpl.java

```diff
@@ -96,8 +96,14 @@ public void deleteProject(MigrationProject migrationProject)
     }
 
     @Override
+    public void deleteOldProvisionalProjects()
+    {
+        this.migrationProjectService.deleteOldProvisionalProjects(0);
+    }
+
+    @Override
     public Long getProjectIdByName(String title){
-        String jql = "SELECT id FROM MigrationProject p WHERE LOWER(p.title) = LOWER(:title)";
+        String jql = "SELECT id FROM MigrationProject p WHERE LOWER(p.title) = LOWER(:title) AND p.provisional = FALSE";
```

Comment:
I think this one is used for project title validation - checking for duplicate titles. I'm not sure if we should check also provisional projects or not...